### ISSUE TITLE
System variable not working at service level

### ISSUE THREAD

**Issue by erandidesilva** [ballerinalang/ballerina#3112]
**Sample code**
```
package httpScenarios;

import ballerina.net.http;
import ballerina.lang.system;

@http:configuration {
    basePath:"/http"
}
service <http> StatusCodeService {
    string connection = system:getEnv("TOMCAT_HOST");

    @http:GET {}
    @http:HEAD {}
    @http:Path {value:"/call/{code}"}
    resource statusCodeResource (message m, @http:PathParam {value:"code"} string codeValue) {
        string resourcePath = "/statusCodeService/codes/";
        message response = {};
        string method = http:getMethod(m);

        system:println(connection);

        resourcePath = resourcePath + codeValue;
        http:ClientConnector httpCheck = create http:ClientConnector(connection);
        response = httpCheck.execute(method, resourcePath, m);

        //int returnCode = http:getStatusCode(response);
        //system:println(returnCode);
        reply response;
    }
}

```
Eventhough "abc" is passed for TOMCAT_HOST, its always taking either GET or HEAD depeneding on the request.

**Comment by anupama-pathirage** [comment]
This will be fixed with #3107

**Comment by anupama-pathirage** [comment]
Fixed with #3107
